aten/src/ATen/native/quantized/cpu/conv_serialization.h

```diff
@@ -169,6 +176,8 @@ ConvParamsSerializationTypeV3 parse_conv_serialized_state(c10::IValue v) {
     for (const auto i : c10::irange(config_a.size(0))) {
       config_vals.emplace_back(config_a[i]);
     }
+    // set default for input_qrange_le_128
+    config_vals[config_vals.size() - 1] = config_vals[config_vals.size() - 1] | (1 << 1);
```

Comment:
Could write this as `config_vals[config_vals.size() - 1] |= (1 << 1);` if you want.